src/pages/Tasks/components/taskComment/CommentItem.tsx

```
@@ -0,0 +1,48 @@
+          </div>
+          {/* Datetime */}
+          <div>
+            <span className="font-mono text-[gray]">
+              {data.createdAt.toLocaleDateString()}
+            </span>
+          </div>
+        </div>
+        <hr />
+        {/* Content */}
+        <div className="ml-2 p-2 text-lg flex items-center justify-between gap-2">
+          <div dangerouslySetInnerHTML={{ __html: data.content }}></div>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)